zulipterminal/ui_tools/buttons.py

```diff
@@ -95,7 +95,7 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
 class HomeButton(TopButton):
     def __init__(self, controller: Any, width: int, count: int=0) -> None:
         button_text = ("All messages   ["
-                       + keys_for_command("GO_BACK").pop()  # FIXME
+                       + keys_for_command("GO_BACK").pop(0)  # FIXME
```

Comment:
`pop` defaults to `pop(-1)` and there's only one element in the set/list right now, so this behavior change should be in the next commit?